SimpleToDo/SimpleToDo/ToDoItem.swift

```diff
@@ -48,6 +48,22 @@ struct ToDoItem: Codable, Identifiable, Hashable {
         }
     }
 
+    /// The value of the item, to convey this information to assistive technologies.
+    var accessibilityValue: String {
+        if isComplete {
+            return "completed"
+        } else {
+            switch priority {
+            case .low:
+                return "low priority"
+            case .medium:
+                return "medium priority"
+            case .high:
```

Comment:
I opted to switch on all the cases here rather than use a `default:` like above. Not sure if there was a reason behind that in the first place?